rllib/agents/trainer.py

```diff
@@ -871,15 +883,30 @@ def evaluate(self) -> dict:
                 raise ValueError("Custom eval function must return "
                                  "dict of metrics, got {}.".format(metrics))
         else:
-            logger.info("Evaluating current policy for {} episodes.".format(
-                self.config["evaluation_num_episodes"]))
+            # How many episodes do we need to run?
+            # In "auto" mode (only for parallel eval + training): Run one
+            # episode per eval worker.
+            num_episodes = self.config["evaluation_num_episodes"] if \
+                self.config["evaluation_num_episodes"] != "auto" else \
+                (self.config["evaluation_num_workers"] or 1)
```

Comment:
Can you make this validation a bit more robust? Can you check if the type is correct?

Reply:
Added proper type/value validation to `Trainer._validate_config`, throwing an error if user sets this to an invalid value.